src/mapbox/poi.js

```diff
@@ -30,6 +30,11 @@ Poi.prototype.getKey = function () {
   return `qmaps_v${version}_favorite_place_${this.id}`
 }
 
+Poi.isPoiCompliantKey = function (k) {
+  const keyPattern = /qmaps_v[0-9]_favorite_place_*/
```

Comment:
You should use the version number from config, it's already imported in this file (import {version} from '../../config/constants.yml')

Reply:
ok for that with RegExp